cryptol-remote-api/python/GettingStarted.md

```diff
@@ -0,0 +1,65 @@
+1. Run the Cryptol remote API server:
+
+    > cryptol-remote-api http /cryptol
+
+This starts an HTTP server with default settings, which means that
+the server can be accessed at `http://localhost:8080/cryptol`
```

Comment:
A general comment: this sentence should be indented three spaces so that it is at the same level of indentation as the "Run the Cryptol remote API server" sentence above. That way, GitHub will render this text with the same indentation as above. Currently, it looks like this:

![cryptol](https://user-images.githubusercontent.com/2364661/203633017-d5060741-81cf-4e2b-9101-7811a3ec19fd.png)

Similar comments apply to all numbered bullet points below this one.